tests/CMakeLists.txt

```diff
@@ -54,6 +54,14 @@ add_test_case(UUIDToString)
 add_test_case(TestIntArrayListToVector)
 add_test_case(TestByteCursorArrayListToVector)
 add_test_case(StringViewTest)
+add_test_case(TestIntArrayListToVector)
+add_test_case(TestByteCursorArrayListToVector)
```

Comment:
`TestIntArrayListToVector` and `TestByteCursorArrayListToVector` are listed twice